src/pages/plugins/index.hbs

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Plugins'
+layout: main.hbs
+---
+
+<div class="container">
+
+  <div class="row">
+    <h3 class="col-sm-12">Plugins</h3>
+  </div>
+
+  <div class="row">
+    <div class="col-sm-12">
+      <p>
+        Here you can find some third-party plugins that can easily extend OpenLayers features.
+      </p>
+    </div>
+  </div>
+  
+  <div class="row">
+    <div class="col-sm-12">
+      <table class="table table-hover">
+        <tbody>
+          <tr><th>Plugin</th><th>Description</th><th>Maintainer</th></tr>
+          <tr>
+            <td><a href="https://github.com/openlayers/ol3-cesium">OL3-Cesium</a></td>
+            <td>Cesium integration library.</td>
+            <td><a href="https://github.com/openlayers">OpenLayers</a></td>
+          </tr>
+          <tr>
+            <td><a href="https://github.com/walkermatt/ol3-layerswitcher">OL3-LayerSwitcher</a></td>
+            <td>Layer control for OpenLayers 3.</td>
+            <td><a href="https://github.com/walkermatt">Matt Walker</a></td>
+          </tr>
+          <tr>
+            <td><a href="https://github.com/walkermatt/ol3-popup">OL3-Popup</a></td>
+            <td>Basic popup overlay for OpenLayers 3.</td>
+            <td><a href="https://github.com/walkermatt">Matt Walker</a></td>
+          </tr>
+          <tr>
+            <td><a href="https://github.com/bjornharrtell/jsts">JSTS</a></td>
```

Comment:
We could make this a list of libraries that integrate well with OpenLayers, not just plugins.